daemon/daemon.go

```diff
@@ -926,14 +926,13 @@ func (d *Daemon) syncLXCMap() error {
 		if added {
```

Comment:
typo in commit: endpoing --> endpoint (you can amend the commit and re-push, but don't have to run tests again for this to be merged)